playbooks/provisioning/openstack/README.md

```diff
@@ -67,6 +67,9 @@ The `openstack_num_masters`, `openstack_num_infra` and
 `openstack_num_nodes` values specify the number of Master, Infra and
 App nodes to create.
 
+The `openstack_node_removal_policies` allows you to specify which App nodes to
+remove.
```

Comment:
Oh, and I'd also like to call it something more obvious, such as: "nodes_to_remove". Ideally, our users shouldn't need to know a ton about openstack and heat.